common/shared.c

```diff
@@ -554,9 +554,6 @@ int ft_getinfo(struct fi_info *hints, struct fi_info **info)
 	if (ret)
 		return ret;
 
-	if (!hints->ep_attr->type)
-		hints->ep_attr->type = FI_EP_RDM;
-
```

Comment:
Some tests assume RDM EP type in the returned fi_info. There was an effort to make tests handle both RDM and MSG EP types. I'm not sure if all tests have been updated so please make sure there no new failures/skipped tests because of this change.